lib/src/Test/Pos/Block/Logic/Mode.hs

```diff
@@ -214,13 +234,28 @@ data BlockTestContext = BlockTestContext
     , btcAllSecrets        :: !AllSecrets
     }
 
-makeLensesWith postfixLFields ''BlockTestContext
+flip makeLensesFor ''BlockTestContext
+    [ ("btcGState", "btcGStateL")
+    , ("btcSystemStart", "btcSystemStartL")
+    , ("btcLoggerName", "btcLoggerNameL")
+    , ("btcSSlottingVar", "btcSSlottingVarL")
+    , ("btcUpdateContext", "btcUpdateContextL")
+    , ("btcSscState", "btcSscStateL")
+    , ("btcTxpMem", "btcTxpMemL")
+    , ("btcTxpGlobalSettings", "btcTxpGlobalSettingsL")
+    , ("btcSlotId", "btcSlotIdL")
+    , ("btcParams", "btcParamsL")
+    , ("btcReportingContext", "btcReportingContextL")
+    , ("btcDelegation", "btcDelegationL")
+    , ("btcPureDBSnapshots", "btcPureDBSnapshotsL")
+    , ("btcAllSecrets", "btcAllSecretsL")
+    ]
```

Comment:
So you did this only for aesthetic reasons? or was there something else? Seems like a bunch of useless changes.

Reply:
I use these lenses for `HasLens blah WalletTestContext blah`. So for my convenience